retorchfiles/scripts/waitforSUT.sh

```diff
@@ -1,19 +1,45 @@
 #!/bin/bash
+
+# Function to check the status of frontend and database
+check_status() {
+  local COUNTER=0
+  local WAIT_LIMIT=20
+  local TJOB_NAME="$1"
+  local DB_SERVICE_NAME="sqldata_$TJOB_NAME"
+  local WEB_SERVICE_URL="http://webmvc_$TJOB_NAME:80"
+  local DB_NAME="[Microsoft.eShopOnContainers.Services.CatalogDb]"
+  local TABLE_NAME="Catalog"
+  local QUERY="USE $DB_NAME; SELECT COUNT (*) FROM $TABLE_NAME;"
+
+  while true; do
+    # Check if frontend service is up
+    if curl --insecure -s "$WEB_SERVICE_URL" | grep -q "<div class=\"esh-catalog-item col-md-4\">"; then
```

Comment:
As you have included a wait for the sut UI in the tests, is this redundant now?

Reply:
@javiertuya 
I added a wait script to my tests but I realized that with it the time measurements are incorrect . However, the code in the test is only temporary. I am currently working on a more comprehensive database waiter in the Set-up @beforall method that creates a connection to the DB and checks that everything is loaded before starting the test.

Reply:
If so, then remove from the tests to allow finish this PR. The priority is to have a stable system, there is time later for enhancements.